master/buildbot/newsfragments/github.feature

```diff
@@ -0,0 +1 @@
+GitHubStatusPush now support reporting to ssh style URLs, ie git@github.com:Owner/RepoName.git
```

Comment:
Could you please add link on GitHubStatusPush, format it with backtricks, and backtricks for `git@github...` (mark it as code). And looks like spell check doesn't like this `RepoName`